main/CMakeLists.txt

```diff
@@ -72,3 +72,6 @@ idf_component_register(
 if( CONFIG_GRI_ENABLE_OTA_DEMO OR CONFIG_GRI_RUN_QUALIFICATION_TEST )
     target_add_binary_data(${COMPONENT_TARGET} "certs/aws_codesign.crt" TEXT)
 endif()
+
```

Comment:
Should this be in the binary when using the DS peripheral?

Reply:
The DS Peripheral is not used in context of the AWS Code Signing certificate verification, so it must be _in_ the binary.